integration-tests/ReadMe.md

```diff
@@ -0,0 +1,17 @@
+# Run integration tests
+
+## Prerequisites
+- A running Fineract instance is required
+
+## Default configuration
+- `backend.protocol=https`
+- `backend.host=localhost`
+- `backend.port=8443`
+- `backend.username=mifos`
+- `backend.password=password`
+- `backend.tenant=default`
+
```

Comment:
Can we name the env variables conforming the existing strategy?

BACKEND_PROTOCOL=https
BACKEND_HOST=...
...